src/client/components/Modal/Modal.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Modal from './Modal';
+
+export default { title: 'Modal' };
+
+export const FeedbackModal = () => <Modal title="Feedback Form" />;
```

Comment:
If you specified `children` as required, why didn't you use it here?
Add it as an example.